react/features/toolbox/actions.native.js

```diff
@@ -283,14 +286,7 @@ export function toggleFullScreen(isFullScreen: boolean): Function {
  */
 export function toggleToolbarButton(buttonName: string): Function {
     return (dispatch: Dispatch, getState: Function) => {
-        const state = getState();
-        const {
-            primaryToolbarButtons,
-            secondaryToolbarButtons
-        } = state['features/toolbox'];
-        const button
-            = primaryToolbarButtons.get(buttonName)
-                || secondaryToolbarButtons.get(buttonName);
+        const button = getButton(buttonName, getState());
```

Comment:
I don't know how I feel about having to pass in state each time but I think it's this whole concept of having to find a button in "primary" and "secondary" that I'm not easy with. As such, and because I know more toolbox work will occur in the future, this seems like a good way to remove duplication.

The PR still looks good to me.